Bio/GenBank/Scanner.py

```diff
@@ -1798,6 +1798,13 @@ def _feed_header_lines(self, consumer, lines):
                                 structured_comment_key is not None
                                 and self.STRUCTURED_COMMENT_END not in data
                             ):
+                                # Don't die on a malformed comment, just warn and carry on
+                                if structured_comment_key not in structured_comment_dict:
+                                    warnings.warn("Structured comment not parsed for %s. Is it malformed?" %
+                                                  consumer.data.name,
+                                                  BiopythonParserWarning, )
+                                    continue
+
```

Comment:
This code snippet is not following the black indentation style, which we have adopted relatively recently in Biopython. Try ``pip install black`` and ``black Bio/GenBank/Scanner.py`` to do this for you (see also the ``CONTRIBUTING.rst`` file). It will give something more like this (I have not checked if this is the exact line breaking):

```python
                                # Don't die on a malformed comment, just warn and carry on
                                if structured_comment_key not in structured_comment_dict:
                                    warnings.warn(
                                        "Structured comment not parsed for %s. Is it malformed?"
                                        % consumer.data.name,
                                        BiopythonParserWarning,
                                    )
                                    continue
```